src/app/code/community/Shopgate/Cloudapi/Block/Customer/Account.php

```diff
@@ -29,4 +29,13 @@ public function getToken()
     {
         return $this->getRequest()->getParam('token');
     }
+
+    /**
+     * @return string
+     * @throws Exception
+     */
+    public function getSgCloudCallbackData()
+    {
+        return $this->getRequest()->getParam('sgcloud_callback_data');
```

Comment:
just because if errors are enabled to display, there will be a warning about echoing ```null``` in the template which may break JS. Use the second param to return an empty string. Do the same for the other function in the class as well.